intranet/apps/eighth/management/commands/dev_generate_signups.py

```diff
@@ -0,0 +1,40 @@
+import datetime
+import random
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand, CommandError
+from django.utils.timezone import localdate, make_aware
+
+from intranet.apps.eighth.models import EighthBlock, EighthScheduledActivity
+
+
+class Command(BaseCommand):
+    help = "Generate 8th Period signups from existing users and activities."
```

Comment:
This would be not good to run in production. The "help" should specify that. Additionally, it would probably be good to add I/O that checks the user means to run this command and warns quite clearly that this shouldn't be run in production. Something like `Are you sure you want to run this command? This is strictly for dev environments and should NEVER be run in production. (y/N)` and maybe even confirm twice before actually running it.